tests/BenchmarkDotNet.Exporters.Plotting.Tests/BenchmarkDotNet.Exporters.Plotting.Tests.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;net462</TargetFrameworks>

```

Comment:
Prefer netstandard2.0 rather than net462.

Reply:
@timcassell merely replicating the Project Settings of BenchmarkDotNet.Tests

Reply:
Oh this is the tests project, my bad. I thought I was looking at the library project.